### ISSUE TITLE
Encoding of OCID Length in draft 14

### ISSUE THREAD

**Issue by huitema** [private-octopus/picoquic#295]
From @nibanks:
I am trying your stateless retry endpoint and my implementation is complaining about your Retry packet:
```
[quic][TX][0] LH Ver:0xd0000ff DestCID:09e4bfe8e7ec21a8 SrcCID:a441cdf1dca781f4 Type:I (Token 0 bytes) (Payload 1207 bytes) (PktNum 1 bytes)
[quic][TX][0]   CRYPTO Offset:0 Len:389
[quic][TX][0]   PADDING Len:797
[quic][RX][null] LH Ver:0xd0000ff DestCID:a441cdf1dca781f4 SrcCID:6883fec7cc5a1c64 Type:R OrigDestCID:09e4bfe8e7ec21a8497f65 (Token 13 bytes)
quic][RX][null] [DROP] Invalid ODCID.
```
Are you using the same CID length for ODCIL encoding as with DCIL and SCIL? (edited)
```
ODCIL:  The length of the Original Destination Connection ID field.
      The length is encoded in the least significant 4 bits of the
      octet, using the same encoding as the DCIL and SCIL fields.  The
      most significant 4 bits of this octet are reserved.  Unless a use
      for these bits has been negotiated, endpoints SHOULD send
      randomized values and MUST ignore any value that it receives.
```

**Comment by huitema** [comment]
Fixed in PR #296

**Comment by huitema** [comment]
Actually not fixed. The new encoding should be applied even when draft-13 is negotiated, because the interop set also includes PR 1498.

**Comment by huitema** [comment]
Finally fixed in PR #297
